simulation_utils/ParameterParser.h

```diff
@@ -46,12 +46,31 @@ class Simulation_Parameters {
     double max_power;
     Magnetorquer MTB; // Magnetorquer class object
 
+    /* Sensors */
+    // GPS
+    double gps_pos_std;
+    double gps_vel_std;
+
+    // Sun Sensors
+    int num_light_diodes;
+    MatrixXd G_ld_b;
+
+    // Magnetometer
+    double magnetometer_noise_std;
+
+    // Gyroscope
+    double gyro_sigma_w;
+    double gyro_sigma_v;
+    double gyro_correlation_time;
+    double gyro_scale_factor_err;
+
     
     /* Simulation Settings */ 
     double MAX_TIME;                   // [s]
     double dt;                         // [s]
     double earliest_sim_start_unix;    // [s]
     double latest_sim_start_unix;      // [s]
+    double sim_start_time;             // [s]
```

Comment:
what epoch is this measured relative to?

Reply:
Done